set.mm

```diff
@@ -85685,6 +85710,201 @@ the first case of his notation (simple exponentiation) and subscript it
       IJK $.
   $}
 
+  ${
+    $d B b f g $.  $d S b f g $.  $d V b g $.
+    $( The class of all functions from a (proper) singleton into ` B ` is the
+       union of all the singletons of (proper) pairs over the elements of ` B `
+       as second component.  (Contributed by AV, 13-Sep-2024.) $)
+    fsetsniunpr $p |- ( S e. V -> { f | f : { S } --> B }
```

Comment:
I think the usual abbreviation for proper pair is `op` following `df-op` rather than `df-pr`

same for fsetabspr and fsetsnprcnex

Reply:
Yes, you are right: I also should write "(proper) ordered pair" to be more precise.

Reply:
~fsetsnprcnex is different: here the "prc" means "proper class"

Reply:
Done